bin/desi_reformat_processing_tables

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import argparse
+
+## Import some helper functions, you can see their definitions by uncomenting the bash shell command
+from desispec.scripts.reformat_proctables import update_processing_tables
+
+
+def get_parser():
```

Comment:
Let's put the parser in desispec.scripts.reformat_proctables.  That's what we do for most other scripts, largely motivated by enabling unit tests to call the parser too.

Reply:
Done